### PR TITLE
Support group identities

### PR DESCRIPTION
**Pull request by Powersource**
I need this to support it in keyring which i need for box2 which i need for tribes2 https://github.com/ssbc/ssb-tribes2/issues/13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "is-canonical-base64": "^1.1.1",
-    "ssb-bfe-spec": "~0.7.0",
+    "ssb-bfe-spec": "ssbc/ssb-bfe-spec#master",
```
Comment (Powersource) [inline package.json:24]: need a new version of ssb-bfe-spec

Comment (staltz) [reply package.json:24]: v0.8.0

Comment (Powersource) [reply package.json]: @staltz updated

**Comment by Powersource** [comment]
feel free to merge and publish

**Comment by staltz** [comment]
Thx!

**Comment by staltz** [comment]
3.7.0

**Pull request merged**